src/test/unit/AllocationManagerUnit.t.sol

```diff
@@ -643,50 +791,44 @@ contract AllocationManagerUnitTests_SlashOperator is AllocationManagerUnitTests
-        uint64 expectedEncumberedMagnitude = allocatedMagnitude - expectedSlashedMagnitude;
-        uint64 maxMagnitudeAfterSlash = WAD - expectedSlashedMagnitude;
-        uint256 slashedStake = DEFAULT_OPERATOR_SHARES.mulWad(expectedSlashedMagnitude);
-        uint256 newSlashableMagnitude = uint256(expectedEncumberedMagnitude).divWad(maxMagnitudeAfterSlash);
+        (uint256 expectedWadSlashed, uint64 expectedCurrentMag, uint64 expectedMaxMag, uint64 expectedEncumberedMag) = _getExpectedSlashVals({
+            magBeforeSlash: allocateParams[0].newMagnitudes[0],
+            wadToSlash: slashingParams.wadsToSlash[0]
+        });
 
         _checkSlashEvents({
             operator: defaultOperator,
             operatorSet: defaultOperatorSet,
-            strategies: defaultStrategies,
-            wadsToSlash: uint256(expectedSlashedMagnitude).toArrayU256(),
-            description: "test"
+            strategy: strategyMock,
+            wadToSlash: expectedWadSlashed,
+            description: "test",
+            currentMag: expectedCurrentMag,
+            maxMag: expectedMaxMag,
+            encumberedMag: expectedEncumberedMag
         });
 
         // Slash Operator
         cheats.prank(defaultAVS);
         allocationManager.slashOperator(defaultAVS, slashingParams);
 
-        // Check storage
-        assertEq(
-            expectedEncumberedMagnitude,
-            allocationManager.encumberedMagnitude(defaultOperator, strategyMock),
-            "encumberedMagnitude not updated"
-        );
-        assertEq(
-            maxMagnitudeAfterSlash,
-            allocationManager.getMaxMagnitudes(defaultOperator, defaultStrategies)[0],
-            "maxMagnitude not updated"
-        );
         _checkAllocationStorage({
             operator: defaultOperator,
             operatorSet: defaultOperatorSet,
             strategy: strategyMock,
-            expectedCurrentMagnitude: expectedEncumberedMagnitude,
-            expectedPendingDiff: 0,
-            expectedEffectBlock: 0
+            expectedAllocation: Allocation({currentMagnitude: expectedCurrentMag, pendingDiff: 0, effectBlock: 0}),
+            expectedMagnitudes: Magnitudes({
+                encumbered: expectedEncumberedMag,
+                max: expectedMaxMag,
+                allocatable: expectedMaxMag - expectedEncumberedMag
+            })
         });
+
+        uint256 slashedStake = DEFAULT_OPERATOR_SHARES.mulWad(expectedWadSlashed); // Wad is same as slashed mag since we start with max mag
         _checkSlashableStake({
             operatorSet: defaultOperatorSet,
             operator: defaultOperator,
             strategies: defaultStrategies,
-            expectedStake: (DEFAULT_OPERATOR_SHARES - slashedStake).mulWad(newSlashableMagnitude)
+            expectedStake: (DEFAULT_OPERATOR_SHARES - slashedStake).mulWad(expectedCurrentMag.divWad(expectedMaxMag))
```

Comment:
What is the mulWad and divWad used here for? 
Is the expectedStake input taking in the expected operatorShares? or the expected slashable operatorShares